### PR TITLE
wallet-ext: fix scrolling in tokens page

### PR DESCRIPTION
**Pull request by pchrysochoidis**
## Description 

* updates layout
* nav menu now is sticky

before

https://github.com/MystenLabs/sui/assets/10210143/b5944d9d-74a0-4357-901e-df3665881729

after

https://github.com/MystenLabs/sui/assets/10210143/0b4af14d-ee86-492f-9e07-c508eeff5ce2

closes [APPS-1706](https://mysten.atlassian.net/browse/APPS-1706)

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/wallet/src/ui/app/index.tsx [hunk 1]
```diff
@@ -159,65 +160,75 @@ const App = () => {
...
+					<Route path="/dapp/*" element={<HomePage disableNavigation />}>
+						<Route path="connect/:requestID" element={<SiteConnectPage />} />
+						<Route path="approve/:requestID" element={<ApprovalRequestPage />} />
+					</Route>
+					{process.env.NODE_ENV === 'development' ? (
+						<Route path="/accounts-dev" element={<AccountsDev />} />
+					) : null}
+				</Routes>
+			)}
+			<div id="overlay-portal-container"></div>
```
Comment (mamos-mysten) [inline apps/wallet/src/ui/app/index.tsx:229]: id probably move these to `src/ui/index.tsx`

**Comment by pchrysochoidis** [comment]
> ran through the UI - noticed one very small issue (may be unrelated to these changes) - but the top border radius seems to be different between pages - but only when using the wallet in the extension window (Extensions > Sui Wallet)
> 
>  Screen.Recording.2023-09-21.at.8.59.19.AM.mov 
> Also - do we still need the `PageLayout` component?

Hmm ok thanks! Let me check!

About `PageLayout` I removed it, (actually changed it to just a div in App), are you referring to something else?

File: apps/wallet/src/ui/app/index.tsx (hunk 1 above)
Comment (mamos-mysten) [reply apps/wallet/src/ui/app/index.tsx:229]: i just meant these two divs

Comment (pchrysochoidis) [reply apps/wallet/src/ui/app/index.tsx]: I ended up moving the whole think there seemed clearer

**Comment by pchrysochoidis** [comment]
fixed the border-radius issue as well


https://github.com/MystenLabs/sui/assets/10210143/25820d68-53e8-4d6c-b85c-1a3c3eb96e2a

**Pull request merged**